wandb/apis/public/runs.py

```diff
@@ -459,6 +485,10 @@ def load(self, force=False):
                     withRuns=False,
                 )
 
+        self._project_internal_id = (
+            int(self._attrs["projectId"]) if "projectId" in self._attrs else None
+        )
```

Comment:
readability nit:

```suggestion
        if "projectId" in self._attrs:
            self._project_internal_id = int(self._attrs["projectId"])
        else:
            self._project_internal_id = None
```

If the line were shorter, a one-liner would be fine, but there is a bit of work in the value and conditional, so it takes 3 lines and most of it is concentrated on one of them.